.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "pynxtools/definitions"]
 	path = pynxtools/definitions
 	url = https://github.com/FAIRmat-NFDI/nexus_definitions.git
+	branch = mpes-fixes
```

Comment:
We should merge this branch and update here to the latest fairmat before merging. So this is just a reminder :)

Reply:
Done!